src/main/java/ch/uzh/ifi/hase/soprafs21/service/DogService.java

```diff
@@ -67,6 +64,7 @@ public void deleteDog(Long ownerId, Long dogId){
         if(optionalDog.isEmpty()) {
             throw new ResponseStatusException(HttpStatus.NOT_FOUND, "No Dog with provided id exists");
         }
+        //TODO check if the dogs owner matches the requester
         this.dogRepository.delete(optionalDog.get());
```

Comment:
Please open a task to fix this for someone so we don't miss it.

Reply:
It is checked a few lines above

Reply:
But that is the userId separately passed with the request, so I think I could post a dog with another owner, and pass my own userId as the ownerId in the request, and it would pass the authorization check since the userId I passed matched the one I am logged in with, even though the dog I passed belongs to another user.

Reply:
Probably, but since the user can only access edit page of his own dogs, I haven't done this check.

Reply:
Removed TODO in commit def59ab

Reply:
I will do the check in the next commit.